src/app/components/tag/TagInput.js

```diff
@@ -106,19 +107,20 @@ class TagInput extends React.Component {
             onKeyPress={this.handleKeyPress.bind(this)}
             placeholder={this.props.intl.formatMessage(messages.search)}
           />
-          <Button
-            style={{ marginLeft: 'auto' }}
+         <Box ml="auto">
+           <Button
             onClick={this.handleSubmit.bind(this)}
             color="primary"
             disabled={this.state.submitDisabled}
           >
-            <FormattedMessage
-              id="tagInput.addTag"
-              defaultMessage="Add Tag"
+           <FormattedMessage
+             id="tagInput.addTag"
+             defaultMessage="Add Tag"
             />
-          </Button>
+           </Button>
+         </Box>
```

Comment:
Please review the indentation in this section, keeping in mind that the indentation unit is 2 spaces, and that closing brackets must be aligned with the line containing the opening tag.

E.g.:

```
          <Box ml="auto">
            <Button
              onClick={this.handleSubmit.bind(this)}
              color="primary"
              disabled={this.state.submitDisabled}
            >
              <FormattedMessage
                id="tagInput.addTag"
                defaultMessage="Add Tag"
              />
            </Button>
          </Box>
```